vendor/bat-native-ledger/src/bat/ledger/internal/contribution/contribution_sku.cc

```diff
@@ -204,37 +204,9 @@ void ContributionSKU::OnGetOrder(
     return;
   }
 
-  auto save_callback = std::bind(&ContributionSKU::TransactionStepSaved,
-      this,
-      _1,
-      std::make_shared<type::SKUOrderPtr>(std::move(order)),
-      callback);
-
-  ledger_->database()->UpdateContributionInfoStep(
-      contribution_id,
-      type::ContributionStep::STEP_EXTERNAL_TRANSACTION,
-      save_callback);
-}
-
-void ContributionSKU::TransactionStepSaved(
-    const type::Result result,
-    std::shared_ptr<type::SKUOrderPtr> shared_order,
-    ledger::ResultCallback callback) {
-  if (result != type::Result::LEDGER_OK) {
-    BLOG(0, "External transaction step was not saved");
-    callback(result);
-    return;
-  }
-
-  if (!shared_order) {
-    BLOG(0, "Order is corrupted");
-    callback(type::Result::RETRY);
-    return;
-  }
-
-  DCHECK_EQ((*shared_order)->items.size(), 1ul);
+  DCHECK_EQ(order->items.size(), 1ul);
```

Comment:
We're no longer using `STEP_EXTERNAL_TRANSACTION` (although it's still present for backward-compatibility), because when the contribution is retried we always need to resubmit the external transaction ID in order to force a status check on the server side.